content/events/2020-london/contact.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Contact
+---
+
+If you’d like to contact us by email: organizers-london@kubernetescommunitydays.org
+
+# Want to be Kept Up-to-date?
+
+Subscribe to the Kubernetes Community Days London mailing list [here](https://mailchi.mp/f5ff97451223/kcdlondon-subscribe)
+
+## Want to Get Involved?
+
+Want to help us build the best Kubernetes conference in London? We're looking for volunteers to help us with organising and day-of logistics. If you think this is something you can help us with, fill out the form [here](https://forms.gle/zKnB6czE25EQkroF8) and we'll be in touch soon!
```

Comment:
@gianarb I'd like to remove this section because we have enough volunteers now. But I want to get this PR merged ASAP because it's been 2 weeks already.